apps/fluent-tester/src/TestComponents/Separator/SeparatorTest.tsx

```diff
@@ -13,26 +15,66 @@ const SeparatorMainTest: React.FunctionComponent = () => {
   return (
     <Stack style={stackStyle} gap={5}>
       <Stack gap={4} style={separatorStackStyle}>
-        <Button content="Button4" />
+        <Button>Button1</Button>
         <BlueSeparator vertical />
-        <Button content="Button5" />
+        <Button>Button2</Button>
         <RedSeparator vertical />
-        <Button content="Button6" />
+        <Button>Button3 </Button>
         <Separator />
       </Stack>
       <Text>This is a text element</Text>
       <Separator />
-      <Button content="This button has longer text" />
+      <Button>This button has longer text</Button>
+    </Stack>
+  );
+};
+
+const CustomisedText = Text.customize({
+  textAlign: 'right',
+  fontSize: 'caption',
+});
+
+const SeparatorAndroid: React.FunctionComponent = () => {
+  return (
+    <Stack style={commonTestStyles.section} gap={5}>
+      <Separator insetSpacing={0} />
+      <CustomisedText>Inset : 0</CustomisedText>
+
+      <Separator insetSpacing={16} />
+      <CustomisedText>Inset : 16</CustomisedText>
+
+      <Separator insetSpacing={56} />
+      <CustomisedText>Inset : 56</CustomisedText>
+
+      <Separator insetSpacing={68} />
+      <CustomisedText>Inset : 68</CustomisedText>
+
+      <Separator insetSpacing={72} />
+      <CustomisedText>Inset : 72</CustomisedText>
+
+      <Separator insetSpacing={108} />
+      <CustomisedText>Inset : 108</CustomisedText>
     </Stack>
   );
 };
 
 const separatorSections: TestSection[] = [
-  {
-    name: 'Basic Button',
-    testID: SEPARATOR_TESTPAGE,
-    component: SeparatorMainTest,
-  },
+  ...Platform.select({
+    android: [
+      {
+        name: 'Basic Mobile Seperator',
+        testID: SEPARATOR_TESTPAGE,
+        component: SeparatorAndroid,
```

Comment:
My personal preference would be to split this test component into its own file and fork the file between Android and non-Android platforms. It reduces the platform checks within the tests and separating the tests out from the main page would be a cleaner organization.

Reply:
Separated to Platform files.  
Just thinking on the platform checks via Platform.select() API , this could be useful some redundant code but also separating helps on the cleaning and making changes and less testing effort. 

But we can have a discussion on this once Ankita is back , As we're also planning to come of common way to handle this.

Reply:
Continuing on another comment, hence resolving this.